src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -145,7 +145,7 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
             services.AddTransient<VirtualFileSystem>();
             services.AddTransient<VirtualFileSystemMiddleware>();
 
-            if (SystemEnvironment.Instance.IsFlexConsumptionSku())
```

Comment:
What was impact with the previous version? Was anything failing?

Reply:
On attempting to migrate linux consumption apps to legion, Yes. 

More details here
https://teams.microsoft.com/l/message/19:6056acddafe94184a21dde3e202b04e1@thread.tacv2/1747919913205?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=201c2bb2-812d-4986-ac92-9bfad0d6cc59&parentMessageId=1747919913205&teamName=Azure%20Functions%20Team&channelName=Release%20-%20Hotfixes%20(proposals%20only)&createdTime=1747919913205